modules/preview-react/segmented-control/lib/hooks/useSegmentedControlItem.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+
+import {
+  createElemPropsHook,
+  composeHooks,
+  useLocalRef,
+  useMountLayout,
+} from '@workday/canvas-kit-react/common';
+import {
+  useListItemRegister,
+  isSelected,
+  useListItemSelect,
+  useOverflowListModel,
+} from '@workday/canvas-kit-react/collection';
+import {useSegmentedControlModel} from './useSegmentedControlModel';
+
+export const useSegmentedControlItem = composeHooks(
+  useListItemSelect,
+  useListItemRegister,
```

Comment:
`useListItemRegister` needs to always be listed last: https://workday.github.io/canvas-kit/?path=/docs/welcome-dev-docs-collections--basic#uselistitemregister
```suggestion
```